boards/acd52832/include/periph_conf.h

```diff
@@ -80,6 +80,11 @@ static const i2c_conf_t i2c_config[] = {
 #define I2C_NUMOF           ARRAY_SIZE(i2c_config)
 /** @} */
 
+/**
+ * @brief Enable the internal DC/DC converter
+ */
+#define NRF5X_ENABLE_DCDC
```

Comment:
What about modelling this with a feature?

Reply:
I got the same question from @aabadie here offline. I think in this case it makes more sense to model it as a define because it is board specific configuration that can always be enabled.

If we model this as a feature the nrf5x MCU's would get a `FEATURES_PROVIDED += NRF5X_DCDC_CONVERTER` and all boards that can use it would require this feature. At this point it doesn't matter anymore whether we model it as a feature or a configuration define and I went for a define here.

This of course assumes that we always want to enable the DC/DC converter when possible, which IMHO makes sense.

Reply:
On sam0 defines are used to for such things, so it's consistent with existing code.